jest.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
+  testURL: "http://localhost/",
```

Comment:
This fixes a localStorage security error with JSDOM - see https://github.com/jsdom/jsdom/issues/2304